### ISSUE TITLE
keep dat address when debugging via deleting the .dat folder

### ISSUE THREAD

**Issue by jcolomb** [datproject/dat#1047]
Hi,
When dat share is not working anymore, the docs tell to erase the .dat folder and run dat share again.

It works, but then the folder gets a different dat address. This is problematic, because
1. I have told people the address of the content
2. the content itself has internal links who are broken when the dat address changes

Is there a way to do the debug while keeping the dat address ? If yes, it would be nice to update the documentation.

**Comment by pfrazee** [comment]
You can't nuke the `.dat` and keep the site. Even if the keys were kept, the lost data would create issues.

When you say "dat share is not working anymore," what's happening?

**Comment by jcolomb** [comment]
note: I had to reinstall nodejs today, and reinstall dat as I was started already.
note 2: it is also supposed to be available on hashbase (dat://rdmpromotion.hashbase.io/)

the terminal looks good:

>  dat v13.11.4
> dat://30241d3351ff12f514e22fead851cc7d047d30439b45c4b2ff1d75cf57f9342b
> Sharing dat: 944 files (68 MB)
> 
> 2 connections | Download 0 B/s Upload 0 B/s
> 
> Watching for file updates


on the beaker browser, It can't find anything, if I look at the network debugger, the peer is found but then dropped

**Comment by pfrazee** [comment]
Huh. Yeah I'm able to create a connection with hashbase to view that dat but I'm not receiving the data. Nothing obvious in the log either ☹️

**Comment by jcolomb** [comment]
strange, the files seem to be accessible here: beaker://library/dat://30241d3351ff12f514e22fead851cc7d047d30439b45c4b2ff1d75cf57f9342b+latest

??

**Comment by pfrazee** [comment]
Yeah this is strange. I'm able to see the file listing and dat.json but
none of the other files. Any chance the original source went down before
those files got uploaded?

**Comment by jcolomb** [comment]
The hashbase version used to  work. I never checked whether the beaker browser seeding was working.
